src/Equinor.ProCoSys.Auth/Authentication/IApiAuthenticator.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Equinor.ProCoSys.Auth.Authentication
+{
+    public interface IApiAuthenticator
```

Comment:
The name seems to indicate something broad in scope, when it is very narrow.
Suggest changing name to indicate the narrow scope, if no other future functionality is planned.
Also strange to have this interface name with a class ApiAuthenticator which does not implement this interface

Reply:
Removed